src/v/security/jwt.h

```diff
@@ -445,9 +432,7 @@ inline result<verifier> make_rs256_verifier(json::Value const& jwk) {
         }
         auto key = crypto::key::load_rsa_public_key(n.value(), e.value());
         return verifier{rs256_verifier{std::move(key)}};
-    } catch (CryptoPP::Exception const& ex) {
-        return errc::jwk_invalid;
-    } catch (crypto::exception const&) {
```

Comment:
We need to keep this, the verifier can still fail, right?

Reply:
whoops!  good catch :)